spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Pooler.scala

```diff
@@ -121,7 +121,7 @@ class Pooler[T: ClassTag] (
       val num_rois = rois.size(1)
       totalNum += num_rois
 
-      if (out.getOrElse(i + 1, null) == null) out(i + 1) = Tensor[T]()
```

Comment:
why delete this? output the output this a table, we can reuse the tensors in the output.